notebooks/official/feature_store/sdk-feature-store.ipynb

```diff
@@ -84,7 +84,8 @@
         "- Create featurestore, entity type, and feature resources.\n",
```

Comment:
<p>Should <code>Type</code> be capital in the title? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/954/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
entity type is actually a correct term. We use it throughout the notebook.